python/src/opendp/domains.py

```diff
@@ -207,6 +208,44 @@ def domain_type(
     return output
 
 
+def expr_domain(
+    lazyframe_domain: Domain,
+    context: Optional[str] = None,
+    grouping_columns: Optional[Any] = None
```

Comment:
Expecting something like `Optional[List[str]]`... but I also wonder if it really needs to be optional: Since it seems to be a list, maybe an empty list can handle that?

Reply:
I added another note to the bootstrap. This is trying to map to the ExprContext enum- if context is set, grouping columns shouldn't be.

Reply:
Looks like it's updated. Thanks!